src/gateway/kong-enterprise/developer-portal/publish-spec.md

```diff
@@ -0,0 +1,28 @@
+---
+title: How to Publish an API Spec to the Dev Portal
+badge: enterprise
+content_type: how-to
+---
+
+Dev Portal provides a single source of truth for all developers to locate, access, and consume services. With intuitive content management for documentation, streamlined developer onboarding, and role-based access control (RBAC), the Dev Portal provides a comprehensive solution for creating and customizing a unified developer experience.
+
+## Publish a Spec to Dev Portal
+
+1. In Kong Manager, navigate to **Dev Portal** > **Editor** and open the link in a new tab. The editor lets you customize the Dev Portal.
```

Comment:
```suggestion
1. In Kong Manager, navigate to **Dev Portal** > **Editor** and open the link in a new tab.
```
Just my opinion, I didn't think the last sentence was necessary for completing the task